core/bindings.rs

```diff
@@ -0,0 +1,88 @@
+use crate::isolate::Buf;
```

Comment:
Can you call this file plugins.rs instead of bindings.rs 

Because we're presenting a specific API rather than access to V8 handles, I think the term is more appropriate. Also it's a nod to an equivalent  browser technology: Netscape Plugins (NPAPI)

Reply:
rename type prefixes as well? I.E. `s/BindingOpResult/PluginOpResult`

Reply:
please